HackIllinois/ViewControllers/HIBaseViewController.swift

```diff
@@ -65,6 +65,46 @@ extension HIBaseViewController {
     }
 }
 
+// MARK: - UIStackView Population
+func populateTagLabels(stackView: UIStackView, tagsString: String) {
+    if stackView.subviews.count != 0 { return }
+    if tagsString.isEmpty { return }
+    let tags = tagsString.components(separatedBy: ",")
+    let padding = "    "
```

Comment:
Why do we need this?

Reply:
tags are a comma separated string since storing string arrays into CoreData broke it, but @sukipat can comment on the others

Reply:
- When dragging the modal view without dismissing it calls viewWillAppear a bunch of times, this just makes sure that it doesn't try to populate it with duplicate tags
- I just didn't want an empty string going through to the default case and making an empty tag, his leaves the stackview empty
- What patrick said
- I didn't want a static width for the tags, padding was the most reliable way to for text to not get cut off by rounded corners

Reply:
Also I added to added to Base because it was called by the cell stackview and the detail stackview and I thought I shouldn't have 30+ lines of a duplicate func

Reply:
Ah cool - feel free to merge